serverless/verify-backup/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN npm ci
 COPY src ./src
 COPY scripts ./scripts
 COPY tsconfig.json ./
+COPY postmangovsg-service-account-key.json ./
```

Comment:
Would avoid doing this as it would bake our credentials permanently into the Docker image and potentially expose it. Another person can just run another command using the built Docker image utilising the included credentials. Might be better if we pass it in from the environment variables instead.

Reply:
Forgot to remove this, the service account key is only needed for local testing. In cloud run, the credentials are auto-detected from the environment. I have removed this now.